Example/Snabble/AppDelegate+AssetProviding.swift

```diff
@@ -44,7 +44,13 @@ extension AppDelegate: AssetProviding {
     }
 
     func url(forResource name: String?, withExtension ext: String?, domain: Any?) -> URL? {
-        Bundle.main.url(forResource: name, withExtension: ext)
+        let url = URL(string: "\(name ?? "")\(ext ?? "")")
+        
+        if let scheme = url?.scheme, scheme == "snabble", let resource = url?.absoluteString {
+            let start = resource.index(resource.startIndex, offsetBy: scheme.count + "://".count)
+            return Bundle.main.url(forResource: String(resource[start...]), withExtension: ext)
+        }
+        return Bundle.main.url(forResource: name, withExtension: ext)
```

Comment:
Was hast du hier genau gemacht und warum?

Reply:
es wurde vorher der Aufruf:

`Bundle.main.url(forResource: "snabble://terms.html", withExtension: nil)`

gemacht um die lokale Resource "terms.html" aufzurufen. Siehe `Onboarding.json:21`:

`"text": "Please accept the [terms](snabble://terms.html) of use and take note of the [privacy](snabble://privacy.html) policy.",`

Die Resource wurde nicht gefunden und es wurde `nil` zurückgegeben. Daraufhin die eine eventuell installierte Snabble App (url scheme: "snabble") aufgerufen.